staging/src/k8s.io/pod-security-admission/cmd/webhook/server/server.go

```diff
@@ -112,6 +112,7 @@ type Server struct {
 
 func (s *Server) Start(ctx context.Context) error {
 	s.informerFactory.Start(ctx.Done())
+	logger := klog.FromContext(ctx)
```

Comment:
This is essentially the main package for the server. You can see where the context is propagated through on line 71. Is there value to migrating this package to contextual logging, if you're not adding a logger to the context?

Reply:
It helps with tracking which code still needs to be converted: if we don't convert server.go, then we need to keep it in the exception list. It's simpler to convert it.

Reply:
@tallclair: are you okay with converting this?

LGTM, but I'll defer to @tallclair. We also still need approval.